core/db/migrate/20160229133259_add_cart_tax_country_iso_to_spree_store.rb

```diff
@@ -0,0 +1,5 @@
+class AddCartTaxCountryIsoToSpreeStore < ActiveRecord::Migration
+  def change
+    add_column :spree_stores, :cart_tax_country_iso, :string, null: false, default: ""
```

Comment:
For adding this field, I would prefer allowing nulls, and setting the default to be null. Then we could validate that the country ISO is within the list of valid ISO's or nil.